D/DuckDB/build_tarballs.jl

```diff
@@ -14,10 +14,6 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/duckdb*/
 
-if [[ "${target}" == *-mingw32 ]]; then
-    sed -i -E "/add_executable\(duckdb_rest_server server.cpp\)$/aif\(\$\{WIN32\}\)\n  set\(LINK_EXTRA -lwsock32 -lws2_32\)\nendif\(\)\n" tools/rest/CMakeLists.txt
-fi
-
```

Comment:
I presume this was made unnecessary by https://github.com/duckdb/duckdb/commit/5f92c916d326a16af70d161a536ed797526c74ae?

Reply:
I think that commit is not part of the 0.3.1 release; this commit is the one I believe addresses the issue: https://github.com/duckdb/duckdb/commit/86307233c4989ff1b538260a0f08ee1bb3fef0ef